.eslintrc

```diff
@@ -1,3 +1,4 @@
 {
   "extends": "airbnb/base",
+  "parser": "babel-eslint"
```

Comment:
Yea, I think my comment is mostly pedantic. We should be fine for a small project like this.